### PR TITLE
[WIP] Fix - recommit ignore insert or remove

### PR DESCRIPTION
**Pull request by heycalmdown**
initial draft

### CODE REVIEW

File: src/transaction.ts [hunk 1]
```diff
@@ -125,35 +125,54 @@ export class Transaction extends events.EventEmitter {
...
         query['$unset'] = query['$unset'] || {};
-        query['$unset']['__t'] = '';
+        query['$unset']['__t'] = '';   
         debug('update recommit query is : ', query);
         if (query._id) {
           query._id = new mongoose.Types.ObjectId(query._id);
         }
-        return collection.findOneAndUpdate({_id: history.oid}, query, function(err, doc) {
-          debug('updated document ', err, doc);
+        return collection.update({_id: history.oid, _t: tid }, query, { w: 1 }, function(err, res) {
```
Comment (heycalmdown) [inline src/transaction.ts]: what is this `_t` for while we are trying to unset the `__t`?

Comment (jiho-kr) [inline src/transaction.ts]: that is typo. changed '_t' to '__t'.

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@types/bluebird": "^3.0.37",
     "@types/debug": "0.0.29",
-    "@types/lodash": "^4.14.50",
+    "@types/lodash": "4.14.56",
```
Comment (kuyh) [inline package.json]: spearhead-ea/island using "lodash": "^4.16.1",

**Comment by kuyh** [comment]
wow it's last year..
Is this commit still valid?

**Pull request closed without merging**